### PR TITLE
Add Getting Started and Aria Optimizations blog

### PR DESCRIPTION
**Pull request by adamjshook**

### CODE REVIEW

**Comment by dborkar** [comment]
@nezihyigitbasi Can you please take a look at Adam's blog on Aria? 
@achopragh @aweisberg FYI

**Comment by aweisberg** [comment]
Thank you for writing this!

Before I get into line level stuff my general feedback is that it is an anti-pattern to put the documentation in the blog. If it's getting started with X material then it should be in the documentation and the blog post can then link to it or paraphrase it.

WDYT?

**Comment by dborkar** [comment]
@aweisberg which doc page should we add it to? 
@mbasmanova would be great if you can take a look too.

**Comment by mbasmanova** [comment]
I'll take a look. My understanding is that this is a copy of an existing blog post: https://datacatessen.com/blog/prestodb-aria/. Instead of copy-pasting, it would be better to just share a link. @aweisberg Ariel, is there a way to do so?

Also, @bhhari is going to make a change to make Aria enabled by default.

**Comment by dborkar** [comment]
@adamjshook content looks good. If you plan to keep the blog on your site, I'd recommend adding a link to it to the top of this post mentioning it is a cross post. 

@mbasmanova As chair of the outreach committee, I am encouraging community members write more content. Adam wasn't aware there was a way to post to prestodb.io, so he first published on his own site.

@bhhari which release does Aria become the default? We can update the blog accordingly.

**Comment by adamjshook** [comment]
@aweisberg I agree that documentation should belong in the docs; I linked to it the source install documentation [here](https://github.com/prestodb/prestodb.github.io/pull/86/files#diff-75e6e9cea158efde426e76747488f52fR17) and [here](https://github.com/prestodb/prestodb.github.io/pull/86/files#diff-75e6e9cea158efde426e76747488f52fR19).  I wanted to keep the inline details as well to demonstrate the simplicity of getting Presto started while keeping the reader on the same page (while linking to the source documentation).  At the time [this page](https://prestodb.io/getting-started.html) did not exist.  I am fine to shorten the blog here up to and including running the docker container and link to this documentation instead.

@mbasmanova I was hoping to cross-post on both sites in order to gain more visibility for the readers.

@dborkar Yes, I can include a link on my site to the post here when it gets published.

**Comment by mbasmanova** [comment]
Facebook's open source folks told me at some point that it is not a good idea to publish multiple copies of the same blog post. I can't remember the reasons though. Ariel, do you remember?

**Comment by adamjshook** [comment]
I was following [this](https://prestodb.io/blog/2020/03/18/uber-pinot) post as an example, which is cross-posted at Uber's engineering blog.  If we'd like to keep it at `prestodb.io` instead of both locations, I can remove it from my site as it will be more useful here.

**Comment by mbasmanova** [comment]
I read the original post and it all looks good to me.

File: website/blog/2020-06-19-getting-started-and-aria.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+author: Adam Shook
+authorURL: https://www.linkedin.com/in/adamjshook
+---
+
+This article was originally published by Adam on June 15th, 2020 over at his blog at [datacatessen.com](https://datacatessen.com/blog/prestodb-aria/).
+
+---
+
+[PrestoDB](https://prestodb.io) recently released a set of experimental features under their Aria project in order to increase table scan performance of data stored in ORC files via the Hive Connector.  In this post, we'll check out these new features at a very basic level using a test environment of PrestoDB on Docker.  To find out more about the Aria features, you can check out the [Facebook Engineering](https://engineering.fb.com/data-infrastructure/aria-presto/) blog post which was published June 2019.
+
```
Comment (aweisberg) [inline website/blog/2020-06-19-getting-started-and-aria.md:12]: Can you add the truncate tag here so it doesn't display the whole post in the blog index?

File: website/blog/2020-06-19-getting-started-and-aria.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+Presto has two core services for executing queries. A _Coordinator_, which is responsible for query parsing and scheduling (among other things), and many _Workers_ which execute the queries in parallel.  The Coordinator can also act as a Worker, though it is not used for production environments.  Since we're playing with Presto here, we'll just use one node to act as both a Coordinator and Worker.  More detailed documentation, including installation details, can be found [here](https://prestodb.io/docs/current/).
+
+Let's take a look at getting a Docker image together for Presto (though they already exist on Dockerhub, e.g. [ahanaio/prestodb-sandbox](https://hub.docker.com/r/ahanaio/prestodb-sandbox)).  We can see below how relatively easy it is to get Presto up and running.  We download Presto, copy some configuration files in a local `etc` directory into the image, and specify an entry point to run the server.
```
Comment (aweisberg) [inline website/blog/2020-06-19-getting-started-and-aria.md:19]: Starting from here I would suggest moving this into the docs under the deployment section

File: website/blog/2020-06-19-getting-started-and-aria.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+CREATE TABLE hive.tpch.part AS SELECT * FROM tpch.sf100.part;
+CREATE TABLE hive.tpch.partsupp AS SELECT * FROM tpch.sf100.partsupp;
+CREATE TABLE hive.tpch.region AS SELECT * FROM tpch.sf100.region;
+CREATE TABLE hive.tpch.supplier AS SELECT * FROM tpch.sf100.supplier;
+
+USE tpch;
+```
+
+Let's talk a bit about how Presto executes a query.  The Presto Coordinator parses the query to build a plan (which we will see examples of below).  Once the plan is made, it is broken into several stages (or fragments) which execute a series of operators.  Operators are a particular function that the engine performs to execute your query.  This typically begins with scanning data via a Connector, then performing operations such as filtering data, partial aggregations, and commonly exchanging data between Presto workers to perform joins and final aggregations.  All of these stages are broken into _splits_, which is a unit of parallelism in Presto.  Workers execute a configurable number of splits in parallel to get your desired results.  All data in the engine is kept in-memory (as long as you don't go past the thresholds of the cluster; another topic for another time).
```
Comment (aweisberg) [inline website/blog/2020-06-19-getting-started-and-aria.md:110]: This is where I would go back to the blog post I think

File: website/blog/2020-06-19-getting-started-and-aria.md [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+│   ├── hive.properties  # Defines configuration properties for our Hive Metastore
+│   └── tpch.properties  # Enables the TPCH connector to generate data
+├── config.properties 	 # Presto instance configuration properties
+├── jvm.config           # JVM configuration for the process
+├── log.properties       # Logging configuration
+└── node.properties      # Node-specific configuration properties
+```
+
+
+The four files directly under `etc` are documented in the above link.  Let's look at the details of the two catalogs: `hive` and `tpch`. The former will be used to read data, and the latter will be used to generate data into our Hive warehouse to run queries.  Setting up Hive is outside the scope of this document, but will be covered in a future post.  In short, you need a catalog service in order to use the Hive connector, such as a Hive Metastore or AWS Glue.  The catalog service contains all of the table definitions used by Presto, and data is typically stored in HDFS or S3.  We've set up a Dockerized Hive Metastore and single-node HDFS instance.
```
Comment (aweisberg) [inline website/blog/2020-06-19-getting-started-and-aria.md:65]: This paragraph would be a good start for a subsection about configuring a connector with TPC-H as an example

File: website/blog/2020-06-19-getting-started-and-aria.md (hunk 2 above)
Comment (aweisberg) [reply website/blog/2020-06-19-getting-started-and-aria.md:19]: FYI when I comment on a paragraph it puts the comment under the paragraph I am commenting on so I am including the paragraph above the comment.

**Comment by adamjshook** [comment]
@aweisberg Thank you for the feedback!  I opened https://github.com/prestodb/presto/pull/14722 to move some of the content over there.  Please let me know your thoughts; I am open to reworking the content or putting it elsewhere.

I will update this PR with the link to the documentation once the other PR is merged (including the `truncate`).

**Comment by aweisberg** [comment]
The docs are merged.

**Comment by adamjshook** [comment]
Thanks, I'll update this PR after the next release so it'll link to the new documentation on prestodb.io

**Comment by aweisberg** [comment]
@adamjshook can you make the changes? Would be great to land this in the next few weeks.

**Comment by adamjshook** [comment]
@aweisberg I was waiting for the next release so the new documentation gets published to https://prestodb.io/docs/current/installation/deployment.html.  Then I can link it properly in the blog post. Thoughts?

**Comment by aweisberg** [comment]
Is this ready for more attention? :-)

**Comment by adamjshook** [comment]
@aweisberg I went to go make the changes to this post, but for the life of me I don't see the Docker example at https://prestodb.io/docs/current/installation/deployment.html even thought it is very clearly in the source [here](https://github.com/prestodb/presto/blob/master/presto-docs/src/main/sphinx/installation/deployment.rst).  I thought I had seen these the other day on the website.  Is something wrong with the build process?

**Comment by adamjshook** [comment]
Oh, I see what is happening now.  They are in the 0.239 docs but since 0.237.2 was deployed after, the 'current' link is pointing to 0.237.2.  https://prestodb.io/docs/0.239/installation/deployment.html

**Comment by adamjshook** [comment]
@aweisberg Please take another look at your convenience.

File: website/blog/2020-08-14-getting-started-and-aria.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+Presto has two core services for executing queries. A _Coordinator_, which is responsible for query parsing and scheduling (among other things), and many _Workers_ which execute the queries in parallel.  The Coordinator can also act as a Worker, though it is not used for production environments.  Since we're playing with Presto here, we'll just use one node to act as both a Coordinator and Worker.  More detailed documentation, including installation details, can be found [here](https://prestodb.io/docs/current/).
+
+We'll use a single Docker container in order to play around with PResto.  An example deployment of a single-node Presto deployment can be found towards the bottom of the deployment documentation [here](https://prestodb.io/docs/0.239/installation/deployment.html).
```
Comment (aweisberg) [inline website/blog/2020-08-14-getting-started-and-aria.md:21]:
```suggestion
We'll use a single Docker container in order to play around with Presto.  An example deployment of a single-node Presto deployment can be found towards the bottom of the deployment documentation [here](https://prestodb.io/docs/0.239/installation/deployment.html).
```

**Comment by aweisberg** [comment]
Can you squash into a single commit? Thanks

**Comment by adamjshook** [comment]
@aweisberg Done!

**Pull request merged**